pkg/nuctl/command/common/renderers.go

```diff
@@ -40,12 +40,7 @@ const (
 	OutputFormatYAML = "yaml"
 )
 
-func RenderFunctions(ctx context.Context,
-	logger logger.Logger,
-	functions []platform.Function,
-	format string,
-	writer io.Writer,
-	renderCallback func(functions []platform.Function, renderer func(interface{}) error) error) error {
+func RenderFunctions(ctx context.Context, logger logger.Logger, functions []platform.Function, format string, writer io.Writer, renderCallback func(functions []platform.Function, renderer func(interface{}) error, withImage bool) error, withImage bool) error {
```

Comment:
Please revert it back to each argument on a different line - it's more readable when you have many arguments.